node/testing/src/scenarios/p2p/pubsub.rs

```diff
@@ -1,16 +1,18 @@
 use std::time::Duration;
 
+use node::ActionKind;
+
 use crate::{
     hosts,
     node::RustNodeTestingConfig,
     scenarios::{ClusterRunner, RunCfg, RunCfgAdvanceTime},
 };
 
-/// Receive a block via meshsub
+/// Receive a message via meshsub
 /// 1. Create a normal node with default devnet config, with devnet peers as initial peers
 /// 2. Wait for 2 minutes
 /// 3. Create a node with discovery disabled and first node as only peer
-/// 4. Wait for first node to broadcast block to second one
+/// 4. Wait for first node to broadcast message to second one
 #[derive(documented::Documented, Default, Clone, Copy)]
 pub struct P2pReceiveBlock;
```

Comment:
change to the comment above replaces "block" with "message", but this is still called `Block`, is that correct?